content/en/continuous_integration/guides/identify_highest_impact_jobs_with_critical_path.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Identify CI Jobs on the Critical Path to reduce the Pipeline Duration
+description: Learn how to identify those CI Jobs that are on the critical path to improve the duration of your CI Pipelines.
+further_reading:
+  - link: "/continuous_integration/search/#pipeline-details-and-executions"
+    tag: "Documentation"
+    text: "Learn how to search and manage your pipeline executions"
+  - link: "continuous_integration/search/#highlight-critical-path"
+    tag: "Documentation"
+    text: "Highlight critical path in your Pipeline Execution"
+---
+
+## Overview
+
+This guide explains how to identify the CI jobs that are on the critical path, with the goal of determining which CI jobs to focus on to reduce the overall duration of the CI pipelines.
+
+### Understanding the Critical Path in a CI Pipeline
+
+The critical path of a CI Pipeline execution is the longest sequence of CI Jobs that determines the total duration of that pipeline execution. In other words, it is the path through the dependency graph of CI Jobs that took the most time to complete. You can only reduce the total duration of a CI Pipeline execution by reducing the duration of CI Jobs that are on the critical path.
+
+{{< img src="continuous_integration/critical_path_highlight_pipeline.png" alt="Highlight of jobs on the critical path in a pipeline execution." width="90%">}}
+
+Looking at the job duration may not be enough. CI jobs are typically executed in parallel with other jobs, meaning that the reduction of the pipeline execution time is determined by reducing the **exclusive time** of the CI job.
+
+The exclusive time of a job on the critical path represents the amount of time the CI runner has spent executing a specific job exclusively, excluding the execution time of other jobs that were running in parallel.
+
+{{< img src="continuous_integration/critical_path_highlight_pipeline_exclusive_time.png" alt="Highlight exclusive time of the jobs on the critical path in a pipeline execution." width="90%">}}
+
+If CI job A is on the critical path with a duration of 100ms and runs in parallel with CI job B, which has a duration of 80ms, the exclusive time of CI job A on the critical path is 20ms. This means that reducing the duration of the CI Job A by more than 20ms would still only decrease the overall pipeline duration by 20ms.
+
+### Supported CI Providers
+
+<div class="alert alert-info">Are you interested in critical path and your CI provider is not supported yet? Fill out <a href="https://forms.gle/zDgiDSGQxA1HgjXh6" target="_blank">this form</a>.</div>
```

Comment:
I would put this after the sentence in which we state for which providers this is enabled, otherwise they don't know 😬